packages/medusa/src/api/routes/admin/auth/delete-session.ts

```diff
@@ -41,6 +41,11 @@
  *    $ref: "#/components/responses/500_error"
  */
 export default async (req, res) => {
-  req.session.destroy()
-  res.status(200).end()
+  if(req.session.customer_id) { // if we are also logged in as a customer, persist that session
```

Comment:
todo: Needs proper formatting
```suggestion
  if (req.session.customer_id) { // if we are also logged in as a customer, persist that session
```